azureDatabaseUtils/src/main/java/bio/terra/workspace/azureDatabaseUtils/runners/PgDumpDatabaseRunner.java

```diff
@@ -0,0 +1,53 @@
+package bio.terra.workspace.azureDatabaseUtils.runners;
+
+import bio.terra.workspace.azureDatabaseUtils.database.DatabaseService;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.boot.ApplicationRunner;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+@Profile("PgDumpDatabase")
+@Component
+public class PgDumpDatabaseRunner implements ApplicationRunner {
+
+  @Value("${env.db.connectToDatabase}")
+  private String sourceDbName;
+
+  @Value("${env.db.url}")
+  private String sourceDbHost;
+
+  @Value("${env.db.port}")
+  private String sourceDbPort;
+
+  @Value("${env.db.user}")
+  private String sourceDbUser;
+
+  @Value("${env.params.dumpfileName}")
+  private String dumpfileName;
+
+  @Value("${env.params.destinationWorkspaceId}")
+  private String destinationWorkspaceId;
+
+  @Value("${env.params.blobstorageDetails}")
+  private String blobstorageDetails;
```

Comment:
what's in blobstorageDetails?

Reply:
this is the blob storage URL with a SAS token included as a query string. WDS makes a couple of WSM API calls to get it (see [here](https://github.com/DataBiosphere/terra-workspace-data-service/blob/main/service/src/main/java/org/databiosphere/workspacedataservice/workspacemanager/WorkspaceManagerDao.java#L50)) - it seems like in this case it should be passed in as an argument, but I'm not certain about that. What do you think?